ContextDesktop/src/main/java/com/github/migi_1/Context/main/Main.java

```diff
@@ -59,20 +59,9 @@ private void setUpMain() {
         settings.setTitle("Carried Away");
         settings.setResolution(1280, 720);
         settings.setVSync(true);
-
-        main.setSettings(settings);
-        VRConfigurer.configureVR(main);
-        main.setPauseOnLostFocus(true);
-    }
-
-    /**
-     * Gets the instance of this Application.
-     *
-     * @return
-     * 		The instance of this Application.
-     */
-    public static Main getMain() {
-    	return main;
+        this.setSettings(settings);
```

Comment:
setSettings seems to be a method in the superClass, consider using super.setSettings(settings) instead of this.setSettings(settings). While it pretty much does the same thing, it makes it a little bit more clear.